src/styles/blocks/container.scss

```diff
@@ -0,0 +1,19 @@
+.container {
+  display: grid;
+  gap: 46px 48px;
+  grid-template-columns: $card-width;
```

Comment:
The grid-template-columns property is set to $card-width, which will create a single column grid. If the intention is to have multiple columns, consider using the repeat() function or specifying multiple column widths directly.